README.md

```diff
@@ -0,0 +1,59 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Phone store
+
+> The phone store application is used to show to the puplic latest phone on market. when the app load, It lists All the latest the phones available in store. user can click on single
+> phone to see its details. the Details of phone includes thone Name, OS, released date and other information
+
+## Built With
+
+- Reactjs
+- Redux
+- CSS
+- react Icons
+- React router DOM
+- API
+
+# [checkout the Demo link](https://latest-phone-store.netlify.app/)
+# [project description](https://www.loom.com/share/15b846d2ef044c4d88a73e96e51a9555)
+
+## Getting Started
+
+**clone the project by running:
+
+```
+$ git clone https://github.com/JoffreyNK/phone-store.git
+
+cd book-store
+
+npm install
+
+npm start
+```
+
+
+## Author
+
+👤 **Joffrey**
+
+- GitHub: [@joffreynk](https://github.com/JoffreyNK)
+- Twitter: [@joffreynk](https://twitter.com/joffreynk)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/joffreynk/)
+
+
+## 🤝 Contributing
+
+Feel free to comment in issues section any bugs or changes that you would like to apply on this project
+check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] It would be better to give some credit to the owner of the design as in the requirements